src/axolotl/utils/config/__init__.py

```diff
@@ -32,7 +33,10 @@ def get_device():
             if torch.backends.mps.is_available():
                 return "mps"
 
-            raise SystemError("No CUDA/mps device found")
+            if is_torch_npu_available():
```

Comment:
@winglian Thanks for your review. I have update the commit so that we use a **variable** instead of a **constant** for getting device. 

BTW, I have a small question here, is the local rank necessary here? Maybe we could replace it with `get_device` in `src/axolotl/utils/distributed.py` to reduce duplicate functions?

Reply:
I think the two `get_device` functions return different things, one returns the device with local rank, the other seems more of a `get_device_type`

Reply:
Got it, thx!